source/speechDictHandler/__init__.py

```diff
@@ -49,6 +52,12 @@ class SpeechDict(list):
 
 	fileName = None
 
+	def create(self, fileName):
+		if os.path.exists(fileName):
+			raise f"can not create dictionary backed by file {fileName}"

```

Comment:
Please raise as an exception type. For a listing of built-in common exceptions see: https://docs.python.org/3/library/exceptions.html

I'd suggest using `FileExistsError`

Reply:
Done